### DISCUSSION TITLE
listPrepDetails, setPrepDetails

### DISCUSSION THREAD

**Discussion by AudibleChocolate** [saleweaver/python-amazon-sp-api#1569] [Feature requests]
Any plans to add the 2 new calls added to Fulfillment Inbound API v2024-03-20 (listPrepDetails, setPrepDetails)?

Thanks for all your work, this has been invaluable.

**Comment by saleweaver** [comment]
Hi

I'm currently working on version 2, which will make it easier to add new methods/adds them automatically. For now, you can either create a PR with the new version, or I'll have a look in a couple of days.

**Reply by AudibleChocolate** [reply]
Awesome. Glad to hear. Thanks for the reply.

**Reply by saleweaver** [reply]
Merged https://github.com/saleweaver/python-amazon-sp-api/pull/1572 yesterday, it's now included thanks to @abrihter
